Demo/Demo/SwiftUIComponents/CardVaultViews/OrderCreateResultView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct OrderCreateResultView: View {
+
+    @ObservedObject var cardVaultViewModel: CardVaultViewModel
+
+    var body: some View {
+        switch cardVaultViewModel.state.createdOrderResponse {
+        case .idle, .loading:
+            EmptyView()
+        case .loaded(let createOrderResponse):
+            getSuccessView(createOrderResponse: createOrderResponse)
+        case .error(let errorMessage):
+            ErrorView(errorMessage: errorMessage)
+        }
+    }
+
+    func getSuccessView(createOrderResponse: Order) -> some View {
```

Comment:
This seems to be the same as the success view for `OrderCompletionResultView` - is there a way we can share some of the similar logic without duplication?